docs/docs-beta/docs/guides/data-modeling/data-assets.md

```diff
@@ -2,3 +2,78 @@
 title: "Create data assets"
 sidebar_position: 10
 ---
+
+Data assets are at the core of Dagster. By modeling your data as assets, you unlock: Declarative Automation, lineage, and a searchable data catalog. This guide will help you understand how to model your data as assets.
+
+There are four ways to create data assets:
+
+* Using the `@asset` decorator to declare a single asset.
+* Using the `@multi_asset` decorator to output multiple assets from a single operation.
+* Using the `@graph_asset` decorator to output a single asset from multiple operations without making each operation itself an asset.
+* Using the `@graph_multi_asset` decorator to output multiple assets from multiple operations
+
+This guide walks you the first three methods. The fourth combines the approaches of the 2nd and 3rd so it's omitted.
+
+***
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+* Dagster installed and a working project setup
+
+</details>
+
+***
+
+## Define a single data asset using the `@asset` decorator
+
+The simplest way to define a data asset in Dagster is by using the `@asset` decorator. This decorator marks a Python function as an asset, with the function's output being the data produced by the asset.
+
+<CodeExample filePath="guides/data-assets/data-assets/asset_docorator.py" language="python" title="Using @dg.asset decorator" />
+
+In this example, my\_data\_asset is an asset that logs its output. Dagster automatically tracks its dependencies and handles its execution within the pipeline.
+
+## Define multiple data assets using the `@multi_asset` decorator
+
+When you need to generate multiple assets from a single operation, you can use the `@multi_asset` decorator. This allows you to output multiple assets while maintaining a single processing function.
+
+<CodeExample filePath="guides/data-assets/data-assets/multi_asset_docorator.py" language="python" title="Using @dg.multi_asset decorator" />
+
+In this example, `my_multi_asset` produces two assets: `asset_one` and `asset_two`. Each is derived from the same function, which makes it easier to handle related data transformations together.
+
+Multi-assets may be useful in the following scenarios:
```

Comment:
Given that this is a guide, I think this section can be cut out -- this is more "concept"-y.